factorial_perm_comp.py

```diff
@@ -0,0 +1,79 @@
+# Script Name		: factorial_perm_comp.py
+# Author			: Ebiwari Williams
+# Created			: 20th May 2017
+# Last Modified		: 
+# Version			: 1.0
+
+# Modifications		: 
+
+# Description		: Find Factorial, Permutation and Combination of a Number
+
+
+def factorial(n):	
+	fact = 1
+	while(n >= 1 ): 
+		fact = fact * n 
+		n = n - 1
+
+	return fact
+
+
+def permutation(n,r):
```

Comment:
Base case can be written to check if r == 0. In this case, simply return 1 instead of calculating the factorial unnecessarily.